### PR TITLE
Hut8/fix logrotate

### PR DESCRIPTION
**Pull request by hut8**

### CODE REVIEW

File: infrastructure/logrotate-nginx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	sharedscripts
+	prerotate
+		if [ -d /etc/logrotate.d/httpd-prerotate ]; then \
+			run-parts /etc/logrotate.d/httpd-prerotate; \
+		fi \
+	endscript
+	postrotate
+		invoke-rc.d nginx rotate >/dev/null 2>&1
+	endscript
+}
```
Comment (joshjordan) [inline infrastructure/logrotate-nginx]: Where did this come from? Did you write it from scratch?

File: infrastructure/provision-rails-assets.yml [hunk 2]
```diff
@@ -196,6 +196,14 @@
...
     - file: path=/etc/nginx/sites-enabled/default state=absent
       notify: restart nginx
+    - name: fix permissions for nginx+passenger+logrotate
+      file: path=/var/log/nginx
+            state=directory
+            mode="u=rwx,g=rx,o=rx"
+    - name: logrotate script
+      copy: src=logrotate-nginx
+            dest=/etc/logrotate.d/nginx
+            mode=0644
```
Comment (joshjordan) [inline infrastructure/provision-rails-assets.yml]: Do we also need something that installs/ensures logrotate is running?

**Comment by joshjordan** [comment]
Two questions for my own education. Merge when ready.

File: infrastructure/logrotate-nginx (hunk 1 above)
Comment (hut8) [inline infrastructure/logrotate-nginx]: It's based off the one that is shipped with the nginx deb package, but I made a bunch of changes. It's much more difficult to use lineinfile

File: infrastructure/provision-rails-assets.yml (hunk 2 above)
Comment (hut8) [inline infrastructure/provision-rails-assets.yml]: Nope. Logrotate comes with the base distro.

Comment (joshjordan) [inline infrastructure/provision-rails-assets.yml]: What keeps it running?

Comment (hut8) [inline infrastructure/provision-rails-assets.yml]: cron

**Pull request merged**